content-audit/api/src/main/java/org/commonjava/propulsor/content/audit/model/FileEvent.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (C) 2011-2018 Red Hat, Inc. (https://github.com/Commonjava/propulsor)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.commonjava.propulsor.content.audit.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.databind.annotation.JsonAppend;
+
+import java.io.Serializable;
+import java.util.Date;
+import java.util.Map;
+import java.util.UUID;
+
+@JsonAppend( attrs = { @JsonAppend.Attr( value = "eventVersion" ) } )
```

Comment:
I think I made a bad decision asking you to change this. If we're going to make decisions about deserializing / using the object, we should probably just have it as a normal field. Then later, I think we can actually use it as a polymorphic type selector if we need to, and refactor into two implementations of the event in order to deserialize two incompatible versions of the class.

Let's switch back to just using a field for the eventVersion, and initialize it to a constant int value of 1.

Reply:
Ack, it's back. :)